test/quic_cfq_test.c

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "internal/packet.h"
+#include "internal/quic_cfq.h"
+#include "internal/quic_wire.h"
+#include "testutil.h"
+
+static const unsigned char ref_buf[] = {
+    0x10, 0x11, 0x12, 0x13, 0x14, 0x15, 0x16, 0x17, 0x18, 0x19
+};
+
+static const uint32_t ref_priority[] = {
+    90, 80, 70, 60, 95, 40, 94, 20, 10, 0
+};
+
+static const uint32_t ref_pn_space[] = {
+    QUIC_PN_SPACE_INITIAL,
+    QUIC_PN_SPACE_HANDSHAKE,
+    QUIC_PN_SPACE_HANDSHAKE,
+    QUIC_PN_SPACE_INITIAL,
+    QUIC_PN_SPACE_INITIAL,
+    QUIC_PN_SPACE_INITIAL,
+    QUIC_PN_SPACE_INITIAL,
+    QUIC_PN_SPACE_INITIAL,
+    QUIC_PN_SPACE_APP,
+    QUIC_PN_SPACE_APP,
+};
+
+static const uint64_t ref_frame_type[] = {
```

Comment:
It seems strange to have an array of all the expected frame types - but then have them all as the same.